### PR TITLE
[stable/goldilocks] deps: bump vpa chart to 2.2.0

### PR DESCRIPTION
**Pull request by aslafy-z**
**Why This PR?**
VPA chart was recently updated

**Changes**
Changes proposed in this pull request:

* Upgrade Goldilocks VPA chart dependency to 2.2.0

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Comment by sudermanjr** [comment]
Hmmm. Since this is a major version change to the VPA chart, we may need to do a major version change for Goldilocks. 

I am curious to see how the tests run though.

File: stable/goldilocks/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "v4.9.0"
-version: 6.8.0
+version: 6.9.0
```
Comment (aslafy-z) [inline stable/goldilocks/Chart.yaml]:
```suggestion
version: 7.0.0
```

**Comment by aslafy-z** [comment]
@sudermanjr FYI, e2e tests passes. I will bump the major so users that overwrites the default vpa values will be notified of the changes.

**Review by sudermanjr — APPROVED** [review]
LGTM

**Pull request merged**